tests/k8s/test_custom_resource_dir.py

```diff
@@ -0,0 +1,48 @@
+from unittest.mock import Mock
+import os
+
+import pytest
+
+from jina.peapods.pods.k8slib.kubernetes_deployment import kubernetes_tools
+from jina import Flow
+
+
+@pytest.fixture()
+def custom_resource_dir(test_dir: str) -> str:
```

Comment:
I think we don't need a fixture for this

Reply:
Removed in https://github.com/jina-ai/jina/pull/3309/commits/e0f6abaa8808d83117fcb0348d39dbb5a7f7a263